.jenkins/pytorch/win-test-helpers/setup_pytorch_env.bat

```diff
@@ -90,6 +90,9 @@ if NOT "%BUILD_ENVIRONMENT%"=="" (
     xcopy /s %CONDA_PARENT_DIR%\Miniconda3\Lib\site-packages\torch %TMP_DIR_WIN%\build\torch\
 )
 
+copy %TMP_DIR_WIN%\build\torch\bin\fbgemm.dll %TMP_DIR_WIN%\build\torch\lib\
```

Comment:
I don't think this is a good fix. If you want to copy them, please write the logic in CMake. Otherwise, I need to fix all the build scripts.

Reply:
Is there any specific reason you don't want to install the DLLs to `lib`?

Reply:
> I don't think this is a good fix. If you want to copy them, please write the logic in CMake. Otherwise, I need to fix all the build scripts.
This is a test to verify the problem. I have submitted a PR to change the install path for 
FBGEMM. https://github.com/pytorch/FBGEMM/pull/296.
But how to handle asmjit.dll ? Which is a third party app.

Reply:
What about adding a install command according to https://stackoverflow.com/a/10000327?

Reply:
Add more cmake install to address this issue in pytorch/FBGEMM#296.